guides/common/attributes-titles.adoc

```diff
@@ -6,6 +6,7 @@
 :AppCentricDeploymentDocTitle: Deploying hosts by using application centric approach
 :ConfiguringLoadBalancerDocTitle: Configuring {SmartProxies} with a load balancer
 :ConfiguringUserAuthenticationDocTitle: Configuring authentication for {ProjectName} users
+:ConfiguringVMSubscriptionsDocTitle: Configuring virt-who for virtual machine subscriptions
```

Comment:
The DocURL for Satellite downstream has to match the title in snake_case, otherwise Pantheon will refuse to build it. Please update it in `attributes-satellite.adoc`

Reply:
Not sure I understand what you mean. Will try.